### PR TITLE
Update book for api gen changes (cl/331666270)

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

**Comment by MarkDaoust** [comment]
@lamberta, @yashk2810

**Review by yashk2810 — APPROVED** [review]

**Review by lamberta — APPROVED** [review]

**Review by ncrubin — APPROVED** [review]

File: docs/_book.yaml [hunk 1]
```diff
@@ -31,7 +31,8 @@ upper_tabs:
       skip_translation: true
       contents:
       - title: "All Symbols"
-        path: /reference/python/openfermion/all_symbols
-      - include: /reference/python/openfermion/_toc.yaml
+        path: /openfermion/api_docs/python/openfermion/all_symbols
```
Comment (yashk2810) [inline docs/_book.yaml:34]: This is the wrong path

Comment (ncrubin) [reply docs/_book.yaml:34]: @lamberta or @yashk2810 can you fix this?

Comment (lamberta) [reply docs/_book.yaml:34]: @MarkDaoust I think this PR can be reverted/closed?

Comment (MarkDaoust) [reply docs/_book.yaml:34]: Yes.

**Pull request closed without merging**